R-package/src/VERSION.txt

```diff
@@ -0,0 +1 @@
+3.0.0-1
```

Comment:
R now has its own version file.

Reply:
What's the reason?

You'll also have to change these places that were using the main LightGBM `VERSION.txt`:

* https://github.com/microsoft/LightGBM/blob/53c64bac52dcbd43505bb3363b00048ecf0e3417/build-cran-package.sh#L24
* https://github.com/microsoft/LightGBM/blob/53c64bac52dcbd43505bb3363b00048ecf0e3417/.ci/test_r_package.sh#L137
* https://github.com/microsoft/LightGBM/blob/53c64bac52dcbd43505bb3363b00048ecf0e3417/R-package/configure.ac#L7

Reply:
R doesn't support `rc`` in version number...
I tried several solutions, and find `-1` can work.
However, using `-1` in python package will be normalized to `post1`, so I use the standalone version number for R.